salesforcecli/status
# Sm/gha

READS via gha
-- can run on demand
-- change schedule from hourly to daily
-- remove circle/dependabot (not used for this repo)

qa:
you can see where I had 0/1 flipped here: https://monitoring.internal.salesforce.com/argusmvp/#/viewmetricsnew?tab=expression&expression=-24h%3A-0m%3Aplatform-cli*%3Aplatform-cli.sfdx.stable.tarball.macos.success%3Aavg%3A1m-avg

and here's the actions run:
https://github.com/salesforcecli/status/actions/runs/3063130034

[if you merge this into main, you'll have a workflow_dispatch button to kick it off]

not tested: failing and posting to slack
